test/integration/kpatch-test

```diff
@@ -152,7 +152,10 @@ build_module() {
 	log "build: $prefix"
 
 	if ! $KPATCHBUILD -n $modname $file >> $LOG 2>&1; then
-		[[ $shouldfail -eq 0 ]] && error "$prefix: build failed"
+		if [[ $shouldfail -eq 0 ]]; then
+			error "$prefix: build failed"
+			cp $HOME/.kpatch/build.log $prefix.log
```

Comment:
The commit msg says, "copy the build log to \<prefix\>.test.log file", so should this have been `cp $HOME/.kpatch/build.log $prefix.test.log` ?